etc/lime-elements.api.md

```diff
@@ -822,7 +823,7 @@ export type EditorTextLink = {
 };
 
 // @beta (undocumented)
-export type EditorUiType = 'standard' | 'minimal';
+export type EditorUiType = 'standard' | 'minimal' | 'no-toolbar';

```

Comment:
Are these mutually exclusive? Should we have a explicit prop to show the toolbar? 🤔

Reply:
see https://github.com/Lundalogik/lime-elements/pull/3530#discussion_r2046974410 

Here, you can only choose one of them at a time. If they were separate boolean props for instance, then they could be mutually exclusive. In that case, if one was true, then the others could not be, because using them together would be logically conflicting or ambiguous.

For example, `no-toolbar` couldn't be true, while `minimal` was true.